.github/workflows/ci.yml

```diff
@@ -32,16 +31,59 @@ jobs:
           ~/.m2/repository
         key: maven-repo
 
-    - name: Initialize CodeQL
-      uses: github/codeql-action/init@v1
+    - name: Build with Maven
+      run: mvn -B verify --file pom.xml
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        USENAME: ${{ github.actor }}
+
+
+  build:
+    if: ${{ $GITHUB_REF == refs/tags/* }}
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
```

Comment:
Please use v2 here.

Reply:
Has been updated. Went for the `zulu` distribution.